docs/topics/i18n/timezones.txt

```diff
@@ -119,8 +123,8 @@ receives one, it attempts to make it aware by interpreting it in the
 :ref:`default time zone <default-current-time-zone>` and raises a warning.
 
 Unfortunately, during DST transitions, some datetimes don't exist or are
-ambiguous. In such situations, pytz_ raises an exception. That's why you should
-always create aware datetime objects when time zone support is enabled.
+ambiguous. That's why you should always create aware datetime objects when time
```

Comment:
I wonder if it's worth documenting what you mean by ambiguous. It is mentioned further down so maybe that's enough.